src/Chainweb/BlockHeaderDB/Internal.hs

```diff
@@ -29,7 +29,6 @@ module Chainweb.BlockHeaderDB.Internal
 (
 -- * Internal Types
   RankedBlockHeader(..)
```

Comment:
Do we still need `RankedBlockHeader` with the `Ranked` newtype? I suppose if we make it wrap a `Ranked BlockHeader`, that will waste some memory by duplicating the height.

Reply:
Yeah, that's the reasoning for not using `Ranked` with `BlockHeader`.